### ISSUE TITLE
OWL ontologies import

### ISSUE THREAD

**Issue by HL-hud** [opencaesar/owl-adapter#58]
## User Story
As a user of OML, I would like to have a way to import existing OWL ontologies within an OML vocabulary file with no need of translation (just as it is under OWL itself).

**Comment by mprather** [comment]
Agree. We have been successful in side-loading an existing OWL file into the process for reasoning and running queries but that is not is not the same as fully integrated scenario where OWL can directly used in OML.
